src/e2e/puppeteer/helpers/dragAndDropThought.ts

```diff
@@ -0,0 +1,35 @@
+import { Page } from 'puppeteer'
+import getEditable from './getEditable'
+
+/** PErforms Drag and Drop functionality on a hought in Puppeteer browser. */
+const dragAndDropThought = async (page: Page, sourceValue: string, destValue: string) => {
+  const sourceElement = await getEditable(page, sourceValue)
+  const destElement = await getEditable(page, destValue)
+
+  const dragStart = await sourceElement.boundingBox()
+  const dragEnd = await destElement.boundingBox()
+
+  if (!dragStart || !dragEnd) {
+    return
+  }
```

Comment:
Since this is in the context of testing, you generally want to `throw` instead of silently returning. Tests should fail hard if any of the preconditions are invalid.